src/envs/behavior_options.py

```diff
@@ -249,7 +243,8 @@ def navigateToOptionPolicy(_state: State,
 
 
 def create_navigate_option_model(
-    plan: List[List[float]], _original_orientation: List[List[float]]
+        plan: List[List[float]], _original_orientation: List[List[float]],
+        _obj_to_nav_to: "URDFObject"
```

Comment:
just curious, how come this argument isn't used? how can we make a navigation option without knowing what we're navigating to?

Reply:
because we already have the plan that was computed by RRT!